packages/application-generic/src/services/auth/auth.service.ts

```diff
@@ -196,32 +193,30 @@ export class AuthService {
   }
 
   @Instrument()
-  async apiKeyAuthenticate(apiKey: string) {
-    const { environment, user, key, error } = await this.getUserData({
-      apiKey,
-    });
+  public async validateApiKey(apiKey: string): Promise<IJwtPayload> {
```

Comment:
Making sure the ApiKey validation returns the same interface as Bearer authentication.